src/sentry/api/endpoints/project_details.py

```diff
@@ -69,6 +69,24 @@
 #: Limit determined experimentally here: https://github.com/getsentry/relay/blob/3105d8544daca3a102c74cefcd77db980306de71/relay-general/src/pii/convert.rs#L289
 MAX_SENSITIVE_FIELD_CHARS = 4000
 
+_options_description = """
```

Comment:
where is this being used?

Reply:
It's [here](https://github.com/getsentry/sentry/pull/56536/files#diff-5999c0ed2577dc8b01a8dc3f0e7bd8749f46d34e06562b9d3582319e59cf85eeR504). I'm trying to avoid cluttering the function b/c the decorator ends up being pretty long and daunting